### ISSUE TITLE
When is this going to be done?

### ISSUE THREAD

**Issue by TheR0b0tMan** [developer-lfierrro743/-Scratch-again-#36]
When?????

**Comment by IceeFoox** [comment]
Likely Never

**Comment by SnesLover22** [comment]
He's a lying dipshit, it'll never be done >:/

**Comment by IceeFoox** [comment]
> He's a lying dipshit, it'll never be done >:/

Likely means it is very possible he wont, its been a long time and its basically ded

**Comment by FailedCoder23** [comment]
# Absolutely Bozo-free
A collection of how I trapped your repetitive Minecraft YT idols in a basement.
<br>
<br>
[![GitHub All Releases](https://img.shields.io/github/downloads/developer-lfierro743/-Scratch-again-/total?style=for-the-badge)](https://github.com/developer-lfierro743/-Scratch-again-/releases)
![Repo size](https://img.shields.io/github/repo-size/developer-lfierro743/-Scratch-again-?style=for-the-badge)
<br/>
# Versions
The following versions are available:

| Version    | Additional Info |
| -------- | ------- |
|Bozo|Stole source code.|
|Website|has -$100, can't buy website|

<br/>

# Usage

` No. `

<br/>

# Information
Was a hoax.

<br/>

</div>

**Comment by FailedCoder23** [comment]
> He's a lying dipshit, it'll never be done >:/

hey we should change the name from eaglery to DGcraft 1.9.4

**Comment by SnesLover22** [comment]
lol

**Comment by IceeFoox** [comment]
DG? as in DeadGamecraft 1.9.4?

**Comment by FailedCoder23** [comment]
darvelgatorcraft

**Comment by FailedCoder23** [comment]
I forked this repo and added some stuff:
https://github.com/FailedCoder23/eaglercraftY

**Comment by SnesLover22** [comment]
lol

**Comment by FailedCoder23** [comment]
snescraft launcher

**Comment by FailedCoder23** [comment]
Literally Never [Latest](https://github.com/developer-lfierrro743/-Scratch-again-/releases/latest)

**Comment by IceeFoox** [comment]
I made that XD

**Comment by IceeFoox** [comment]
Btw, yeah, there is a DMCA takedown notice on Eaglercraft 1.12

**Comment by SnesLover22** [comment]
oop :/

**Comment by IceeFoox** [comment]
I got an email from them

**Comment by FailedCoder23** [comment]
> I got an email from them

so did i

**Comment by SnesLover22** [comment]
yeah they're freaking out about the revolt :/ download the contents guys, quickly.

**Comment by IceeFoox** [comment]
How?

**Comment by SnesLover22** [comment]
fork the repo or something, or click download by clicking code section, then click the green code button, and download the zip.

**Comment by SnesLover22** [comment]
Yeah but they fucking mad about my revolt on reddit I think :/

**Comment by FailedCoder23** [comment]
lol the asshole changed the README

**Comment by FailedCoder23** [comment]
https://github.com/developer-lfierrro743/-Scratch-again-/discussions/40

**Comment by FailedCoder23** [comment]
@developer-lfierrro743 here's your fucking source code:
[sauce code realz oh my god??](https://bit.ly/3ZZ3sgd)

**Comment by TheR0b0tMan** [comment]
@SnesLover22 where is the repo for 1.12
